designs/administrator/olm/convey-installing-operator/index.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Convey Operator is Installing
+
+Currently when an operator is installed the user is taken to the **Installed Operators** list view, where it may take some time before the entry for the newly installed operator actually appears, particularly if the operator is installed globally and the user is viewing a namespace the operator will be copied to, and the CSV doesn't yet exist. This design gives the admin user immediate feedback that the operator is installing properly, and can surface errors that may occur during the installation process.
+
+## Installing an Operator
+
+![Install operator](img/1-1-install.png)
+- The user selects the operator they'd like to install and selected installation options (no changes to this screen.)
+
+![Installing operator](img/1-2x-installing-leave.png)
+- The user is now taken to the new installing screen where a spinner conveys the operator is currently being installed.
+- The status conditions reason and phase are shown and update in real time on the installing screen.
+- A link is included that allows the user to exit to the Installed Operators list.
+- The user could navigate away from this screen using the left nav, and return to it via the **Installed Operators** list view.
+
+![Installed operator list](img/y-1-list.png)
+- On the Installed Operators list, a row item would always appear (even if the CSV isn’t created in that project yet) that conveys the operator is installing.
+- A **View Install** link will be present when the operator is installing, that can return the user to the installing screen.
+- **Definition of 'operator installing':** Before CSV's status.phase reaches "Succeeded"
+- Values in columns may be unfilled as neccessary.
+
+![Installing operator](img/1-2-installing.png)
+- The user would return to the installing screen.
+
+![Installing operator complete](img/1-3-complete.png)
+- When installation completes successfully, the user has the ability to view the installed operator (or view the installed operator list.)
+
+![Operator details](img/x-1-operatordetails.png)
+- The user views the details of the operator.
+
+## Warnings and Errors
+
+![Manual approval required](img/1-5-manualApproval.png)
```

Comment:
@itsptk Do we only show the resource list when there's an approval for some reason? Or should we pull this list out of the screenshot since we will be trying to surface resources including RBAC things as a separate story?  I wasn't sure if it was really tied to this flow or not.

Reply:
Yeah this is just sort of moving the existing 'review install plan' behavior into this screen. Today when you install an operator with manual approval it just gets stuck in that 'updating' state that we are trying to solve with https://issues.redhat.com/browse/PD-458 and it takes a few steps to navigate to the Install Plan where you see this resource list. Basically today the same (not ideal) interaction is used whether its the first install of the operator or an update, whereas this design is now saying that for the install case the approval should just be moved into this flow. The design in PD-458 will improve the case where an already-installed operator is ready to update and needs approval.

This was needed in this story since it affected the part 1 of this series of epics (just conveying install), and it wasn't clear when we'd get to part 3 (show resources during install.) My understanding is that showing the list of resources any earlier than this in the flow would still require work from OLM to surface the list of to-be-installed up to the console (somehow generate that list early.)

Curious if you agree with this thinking @tlwu2013